### ISSUE TITLE
Reproduce the result in Supplementary Table 1

### ISSUE THREAD

**Issue by Yu979** [fteufel/signalp-6.0#12]
Hi, I found it extremely slow and also problematic when I ran train_model.py with the basic training command (shown in the figure attached), and I want to reproduce the result in your Supplementary Table 1, could you please tell me how I can do it?
![image](https://user-images.githubusercontent.com/55450247/206365186-0efdd97a-ac31-4fec-bd77-590929ad2f84.png)

**Comment by fteufel** [comment]
Hi, could you share the loss curve? Maybe the learning rate isn't working, hard to tell from the epoch logs.

**Comment by Yu979** [comment]
Hi, this is my loss curve for 30 epochs.
![Train_loss](https://user-images.githubusercontent.com/55450247/207777726-a5e6a09a-baf1-4f8d-ab38-ada81d826644.png)

**Comment by Yu979** [comment]
And this is for 140 epochs
![Train_loss](https://user-images.githubusercontent.com/55450247/207826031-280c7a74-7788-4ceb-ae26-1eb3a1f95ee0.png)

**Comment by fteufel** [comment]
I think the learning rate here is too high.
```
train_model.py --data data/train_set.fasta --test_partition 0 --validation_partition 1 --output_dir testruns --experiment_name testrun1 --remove_top_layers 1 --kingdom_as_token --sp_region_labels --region_regularization_alpha 0.5 --constrain_crf --average_per_kingdom --optimizer adamax --lr 0.000008
```
shows performance after one epoch.

**Comment by fteufel** [comment]
Unfortunately there is no code for this. Most of these tools are only available via an online submission interface, so you will have submit the data to the services and parse their outputs.
